### ISSUE TITLE
Missing fields in Refund object

### ISSUE THREAD

**Issue by Flo354** [stripe/stripe-java#620]
Hi there,

According to the documentation https://stripe.com/docs/api/refunds/object, there are two fields: `failure_balance_transaction` and `failure_reason`.

From this same link, I have an example:
```
com.stripe.model.Refund JSON: {
  "id": "re_1DZjdEGdgZUYsmXt12ITarBV",
  "object": "refund",
  "amount": 1000,
  "balance_transaction": "txn_1DZjdEGdgZUYsmXtzUZtSJP3",
  "charge": "ch_1DZjcVGdgZUYsmXtPWVFehWQ",
  "created": 1542998640,
  "currency": "eur",
  "failure_balance_transaction": "txn_1DZjdHGdgZUYsmXtwGer6WKP",
  "failure_reason": "expired_or_canceled_card",
  "metadata": {
  },
  "reason": null,
  "receipt_number": null,
  "source_transfer_reversal": null,
  "status": "failed",
  "transfer_reversal": null
}
``` 

In the SDK these fields are missing. Furthermore, in the SDK there is a field called `description` but it is not in the documentation.

Am i missing something, or is it an error?

EDIT : there are other fields missing like `transfer_reversal`

**Comment by ob-stripe** [comment]
Fixed in 7.10.0.

**Comment by Flo354** [comment]
Thanks @ob-stripe.

When will the release be pushed to maven?

**Comment by ob-stripe** [comment]
Hey @Flo354, sorry about that, looks like something went wrong during the release process.

I've just released stripe-java v7.10.0 on OSS Sonatype. It should appear on Maven Central within a day.
